Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.9
+LABEL org.opencontainers.image.authors="Pbench Maintainers <pbench@googlegroups.com>"
+
+WORKDIR /var/tmp
+
+RUN python3 -m pip install --upgrade pip
+
+# Since we install the relay file in/from /src, we seem to need to include it
+# in the PYTHONPATH.
+# FIXME:  how do we get this to install conventionally so we don't need this definition?
+ENV PYTHONPATH=/src/file-relay/relay/
```

Comment:
Previously, I've used `python3 -m pip install --edit .` to install it. (I wasn't able to make it work without the `--edit`, for whatever reason.) This does put a `relay` click wrapper in `~/.local/bin/relay`, which is what I used in the service file to start the relay server on the staging VM.

Arguably we should have "real" pip packaging rather than relying on the in-place `--edit` setup... but after the initial install, the service file seems to work fine without setting `PYTHONPATH` manually.

```
ExecStart=/home/pbench/.local/bin/relay --server_id RELAY --bind 10.1.170.231:8002 --files-directory /srv/RELAY
```

Reply:
OK, the problem seems to be that the package/wheel/etc. that we build creates the executable stub but doesn't incorporate the relay sources!  So, when I install it, I get nothing in `/usr/local/lib/...` except the `.dist-info` file.  😬

Reply:
OK, so I reorganized the directory tree layout in the Git checkout (see #8), and now the packaging seems to work as expected.  So, a `pip install .` (with our without the `-e`) produces the desired effects, and an explicit `PYTHONPATH` definition is no longer required.

Reply:
However, I now need to remove the definition (and the `FIXME`) that you cited....

Reply:
Done!